Jenkinsfile

```diff
@@ -17,11 +17,12 @@ pipeline {
       }
       steps {
 	sh 'python3 -c "import torch; torch.cuda.current_device()"'
+	sh 'python3 -c "import tensorflow as tf; tf.test.is_gpu_available()"'
```

Comment:
Can I get a brief explanation for my own education: Is this line here to get a diagnostic, an error message if gpu is not available, but the tests are still attempted, or does failure at this stage entail the script being stopped? (Essentially I am asking the equivalent of "Is `set -e` to be assumed or not")

Reply:
It was mostly there to force a failure if TF wasn't working properly. Otherwise we'd just get segfaults in the middle of the unit tests without much explanation. It's not strictly necessary but acts as a useful canary.